tests/test_vsac.py

```diff
@@ -34,10 +34,11 @@ def test_download_oid_data(mock_api, mock_db, name, umls, force, raises, tmp_pat
         )
         vsac.download_oid_data(name,path=tmp_path, config=config)
         output_dir = list(tmp_path.glob("*"))
-        assert len(output_dir) == 5
+        print(output_dir)
+        assert len(output_dir) == 4
```

Comment:
Why not just check the actual filenames? Those feel like we'd want to notice a change there too, not just in the number of files.

Reply:
sure, can do.